extension/src/telemetry/index.ts

```diff
@@ -31,6 +31,8 @@ export const getTelemetryReporter = (): TelemetryReporter => {
   return telemetryReporter
 }
 
+type EventName = keyof IEventNamePropertyMapping
```

Comment:
@sroy3 @wolmir @julieg18

[F] Upgrading to Typescript `4.8.2` caused the compiler to fail on a couple of things in this file.

Using this instead of extending the types actually makes this code easier to read IMO.

Let me know if you'd like to see something different.